lib/bundler/dsl.rb

```diff
@@ -107,7 +107,8 @@ def gem(name, *args)
           unless deleted_dep
             return if dep.type == :development
             raise GemfileError, "You cannot specify the same gem twice with different version requirements.\n" \
-                            "You specified: #{current.name} (#{current.requirement}) and #{dep.name} (#{dep.requirement})"
+                            "You specified: #{current.name} (#{current.requirement}) and #{dep.name} (#{dep.requirement}). " \
+                            "You can also update the gem by running `bundle update #{current.name}`"
```

Comment:
Let's change the wording to be more clearer... saying "You can also update..." leaves questions as to what are the other solutions to "also".
I see a couple of error cases - the users intent could be to update the gem version or might have forgotten it was already specified. I think saying something like this would be better.

```
"You cannot specify the same gem twice with different version requirements.\n" \
"You specified: #{current.name} (#{current.requirement}) and #{dep.name} (#{dep.requirement}). " \
"If you want to update the gem version, run `bundle update #{current.name}`. You may need to change the version requirement specified in the Gemfile if it's too restrictive."
```

Reply:
It would be great if you can add tests for this. Also check if this error is triggered in other use cases like if a gem was specified twice in a Gemfile and a user runs `bundle install`. The error should still makes sense for those use cases.